app/styles/crate.scss

```diff
@@ -313,6 +313,13 @@
         pre {
             overflow-x: auto;
         }
+
+        p {
+            code {
+                background-color: #fff;
```

Comment:
Could you edit your commit message to be more clear on what this change is affecting?